Dockerfile

```diff
@@ -40,14 +40,20 @@ RUN apt-get update && apt-get install -y \
 	createrepo \
 	curl \
 	dpkg-sig \
+	e2fslibs-dev \
 	gcc-mingw-w64 \
 	git \
 	iptables \
 	libapparmor-dev \
 	libcap-dev \
 	libsqlite3-dev \
+	liblzma-dev \
+	libxml2-dev \
```

Comment:
@LK4D4 this is somewhat complicated. libxml2-dev contains both static and dynamic libraries, and depending on how Docker is build (binary vs dynbinary), either static or dynamic libxml2 is used.

Now, when libxml2 is statically linked into a library, the linker complains like this
> warning: Using 'getaddrinfo' in statically linked applications requires at runtime
> the shared libraries from the glibc version used for linking

This is glibc-specific thing, they choose to make resolver always dynamic. The warning is totally harmless as we don't use http functionality in libxml2. Nevertheless, I chose to compile a minimal version of libxml2 (only static one) for the case of static linking to avoid these warnings. Some more details are at https://github.com/kolyshkin/docker/commit/358c3034064156ed28c32342c79f4661d15f0572

So, libxml2-dev (and libz and liblzma) packages are used for "dynbinary", while the one I compile in Dockerfile is used for "binary" (static linking).

I understand it's messy; my plan is to try to get rid of libxml2 dependency altogether.